spec/features/user_can_signup_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'registration' do
```

Comment:
This is an aesthetic thing but standards are standards...

When running tests through capybara we prefer to use `feature` over `describe` and `scenario` over `it`. Again, this will work, it's just being fussy. Similar to how we use `snake_case` for ruby and `camelCase` for javascript even though they're not necessary.

Have a look at this [fairly easy to read bit of documentation](https://relishapp.com/rspec/rspec-rails/v/3-9/docs/feature-specs/feature-spec).

Reply:
I would do something like:

```ruby
 feature "User registration"
	scenario "with valid credentials will redirect the user to the home page"
```